tsconfig.jest.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Tohle můžeme celé zahodit, prosím, v `package.json` se odkazujeme na `tests/tsconfig.jest.json`. (Přestěhoval jsem ten soubor, aby nestrašil v rootu, a nedal o tom vědět – pardon.)

Reply:
O tom vím, jen to uteklo v rebase, odstraněno v 1ffbbb8.